pkgs/shells/mrsh/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkgconfig, readline }:
+
+stdenv.mkDerivation rec {
+  pname   = "mrsh";
+  version = "ef21854fc9ce172fb1f7f580b19a89d030d67c65";
+
+  src = fetchFromGitHub {
+    owner = "emersion";
+    repo = "mrsh";
+    rev = version;
```

...and specifying the commit hash only in `fetchFromGitHub`.

```suggestion
    rev = "ef21854fc9ce172fb1f7f580b19a89d030d67c65";
```